provision.yaml

```diff
@@ -20,3 +23,6 @@ travis_el7:
 release_checks:
   provisioner: vmpooler
   images: ['redhat-5-x86_64', 'redhat-6-x86_64', 'redhat-7-x86_64', 'redhat-8-x86_64', 'centos-5-x86_64', 'centos-6-x86_64', 'centos-7-x86_64', 'centos-8-x86_64', 'oracle-6-x86_64', 'scientific-6-x86_64', 'scientific-7-x86_64', 'debian-8-x86_64', 'debian-9-x86_64', 'debian-10-x86_64', 'ubuntu-1404-x86_64', 'ubuntu-1604-x86_64', 'ubuntu-1804-x86_64']
+release_checks:
```

Comment:
If you want to separate the jobs for Litmus acceptance you can split the release checks for puppet 5/6 if needed, but if you are using the same machines you can keep a single group, either way you will need to clean up this provisioning file.